style.css

```diff
@@ -0,0 +1,633 @@
+  }
+
+  .main-page {
+    margin: 0;
+  }
+
+  .top-bg {
+    background: #272a31;
+  }
+
+  .fa-brands {
+    color: #fff;
+  }
+
+  .top {
+    display: flex;
+    color: #fff;
+  }
+
+  .lg-logo {
+    display: flex;
+    padding: 0 10px;
+  }
+
+  .logo {
+    padding: 5px;
+    font-weight: 700;
+    display: block;
+    text-decoration: none;
+    color: #272a31;
+    text-align: center;
+  }
+
+  .list {
+    display: flex;
+    gap: 20px;
+    list-style: none;
+    font-weight: 300;
+    color: #272a31;
+    padding: 5px;
+    align-items: center;
+  }
+
+  .list-li {
+    color: #ec5242;
+    list-style: none;
+    text-decoration: none;
+  }
+
+  .btn {
+    border: 2px solid #ec5242;
+    color: #ec5242;
+    display: flex;
+    font-size: 15px;
+    font-weight: 500;
+    padding: 5px;
+    text-decoration: none;
+  }
+
+  .nav-1 {
+    gap: 5%;
+    padding: 0;
+    margin: 0;
+  }
+
+  .first {
+    font-weight: 400;
+    padding-top: 10px;
+    color: #ec5242;
+    margin: 0;
+  }
+
+  .second {
+    font-weight: 900;
+    color: #ec5242;
+    margin: 20px;
+    font-size: 25px;
+    margin-bottom: 15px;
+  }
+
+  .second-pr {
+    margin: 20px;
+    font-weight: 400;
+    color: #272a31;
+    margin-bottom: 15px;
+  }
+
+  .dates {
+    font-weight: 700;
+    margin: 20px;
+  }
+
+  .menu {
+    cursor: pointer;
+    align-self: center;
+    display: none;
+  }
+
+  .bar {
+    width: 15px;
+    margin: 4px auto;
+    background-color: black;
+    height: 2px;
+    display: none;
+  }
+}
+
+.card-section {
+  background-color: #272a31;
+  margin: 0;
+  padding: 20px 0;
+}
+
+.card-main {
+  align-content: center;
+  color: #fff;
+  text-align: center;
+  margin: 0;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 6px;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.cd-card {
+  display: flex;
+  gap: 20px;
+  background: #6c757d;
+  width: 100%;
+  align-items: center;
+}
+
+.fa-solid {
+  font-size: 30px;
+  width: 10%;
+}
+
+.fa-brands {
+  font-size: 30px;
+  width: 10%;
+}
+
+.cards-hd {
+  color: #ec5242;
+  font-size: 15px;
+  width: 30%;
+}
+
+.cards-pr {
+  color: #fff;
+  font-size: 2px;
+  font-weight: 100;
+  width: 55%;
+}
+
+.main-btn-2022 {
+  text-align: center;
+}
+
+.card-btn {
+  color: #fff;
+  text-align: center;
+  display: none;
+}
+
+.btn-2022 {
+  background: #ec5242;
+  padding: 10px;
+}
+
+@media (min-width: 768px) {
+  .card-section {
+    margin: 0;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .cd-card {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    width: 18%;
+    padding-top: 10px;
+  }
+
+  .btn-2022 {
+    display: none;
+  }
+}
+
+.speakers-section {
+  background-color: #fff;
+}
+
+.spks-main {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers {
+  display: flex;
+  margin: 0;
+  align-items: center;
+  padding: 0;
+  flex-direction: row;
+  gap: 14px;
+  width: 100%;
+}
```

Comment:
- [ ] As requested by the previous reviewer, Kindly align the content in the center of the page and make sure to use the same typography in order to look like the provided design. :wink: 

- [ ] Kindly make sure the `Featured Speakers` title is still visible.
   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/204110522-ddb40531-3b64-4a80-b05e-0ec26992c387.png)


   - ### Provided design
   ![image](https://user-images.githubusercontent.com/87197412/204110532-f1106d13-c3a7-4093-8b42-8104d4532cb2.png)